pkg/controller/route/route_controller.go

```diff
@@ -110,11 +146,7 @@ func (rc *RouteController) Run(stopCh <-chan struct{}, syncPeriod time.Duration)
 		rc.broadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: rc.kubeClient.CoreV1().Events("")})
 	}
 
-	// TODO: If we do just the full Resync every 5 minutes (default value)
```

Comment:
You have removed the TODO: about resyncing every 5 minutes. However the code still resyncs every 5 minutes has been left in. This sort of resync is an anti-pattern we would like to remove. We should only be doing this sort of resync if we are trying to align with an external system that requires it. Please leave the TODO in if you are leaving in the resync.

Reply:
Ok, I will leave the `TODO` @cheftako